Test level

fireburningblue opened the pull request:
Removed walls, added kill z, and added character model

NateBellcockOC committed:
Merge branch 'Replaction' into programmers

NateBellcockOC committed:
Merge branch 'programmers' of https://github.com/EpicPinkCreeper/teamgameproject1 into programmers

fireburningblue committed:
Update .gitignore

Added more files to .gitignore

fireburningblue committed:
Changing Level Design

Removed the walls, organized the files, and added the character

NateBellcockOC reviewed (changes requested):
Kill box works but when I respawn I am off the map in a Conner and fall into the kill box again. please move respawn to land.

fireburningblue committed:
Issue Fix

Fixed issue where player spawned off platform. New issue, need permission to work on Character Movement so that spawn location can change.

fireburningblue commented:
Issue Fixed, requesting verification.

NateBellcockOC reviewed (approved):
The respawn issue has been fixed
